### ISSUE TITLE
chore: update the remaining trackEvent calls to use new builder pattern

### ISSUE THREAD

**Issue by NicolasMassart** [MetaMask/metamask-mobile#12262]
### What is this about?

Followup on #12180 that updated the metrics trackEvent and some files calling it, but not all.
Finish the update of the files that are using trackEvent in the old way to use the new one with builder.

### Example current code
```ts
const { trackEvent } = useMetrics();
...
trackEvent(MetaMetricsEvents.ACTIONS_BUTTON_CLICKED, {
  text: '',
  chain_id: getDecimalChainId(chainId),
});
```
### Example target code
```ts
const { trackEvent, createEventBuilder } = useMetrics();
...
trackEvent(createEventBuilder(MetaMetricsEvents.ACTIONS_BUTTON_CLICKED)
      .addProperties({
      text: '',
      chain_id: getDecimalChainId(chainId),
    }).build()
  );
```

### Acceptance Criteria

- all files using trackevent updated
- all unit test files updated and tests passing

#### Files list
The list of the files to update is the following (some may be already fixed after I write this list)

see https://github.com/MetaMask/metamask-mobile/pull/12180#issuecomment-2471385209

### Stakeholder review needed before the work gets merged

- [X] Engineering (needed in most cases)
- [ ] Design
- [ ] Product
- [ ] QA (automation tests are required to pass before merging PRs but not all changes are covered by automation tests - please review if QA is needed beyond automation tests)
- [ ] Security
- [ ] Legal
- [ ] Marketing
- [ ] Management (please specify)
- [ ] Other (please specify)

### References

fix: trackevent enabled is undefined #12180

**Comment by NicolasMassart** [comment]
Note, some of the files listed above may already be fixed in #12180

**Comment by NicolasMassart** [comment]
Closed as work was continued in the original PR
